rpc/clients/typescript/src/ws_client.ts

```diff
@@ -278,17 +279,19 @@ export class WSClient {
      * @param perPage number of signedOrders to fetch per paginated request
      * @returns all orders, their hash and their fillableTakerAssetAmount
      */
-    public async getOrdersAsync(perPage: number = 200): Promise<OrderInfo[]> {
+    public async getOrdersAsync(perPage: number = 200): Promise<GetOrdersResponse> {
```

Comment:
This is a breaking change so we need to bump the next release number up a major version.